### PR TITLE
Added support for Qianmo.com

### PR DESCRIPTION
**Pull request by taichatha**
This should work! I tested on a few videos, and it reads the video correctly and downloads them.

### CODE REVIEW

File: youtube_dl/extractor/qianmo.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# http://qianmo.com/bazingayue/324
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from lxml import html
+import requests
```
Comment (dstftw) [inline youtube_dl/extractor/qianmo.py]: These dependencies are unacceptable. Moreover you don'y use them.

File: youtube_dl/extractor/qianmo.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+class QianmoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?qianmo\.com/(?P<user>[A-Za-z]+)/(?P<id>[0-9]+)'
+    _TEST = {
+        'url': 'http://qianmo.com/kawaei/281',
+        'info_dict': {
+            'id': '281',
+            'ext': 'mp4',
+            'title': '[中字] Google新logo宣传片 「Google, 进化」 - 阡陌视频社区',
+            # TODO more properties, either as:
```
Comment (dstftw) [inline youtube_dl/extractor/qianmo.py]: Either **do** or remove.

File: youtube_dl/extractor/qianmo.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+        title = self._html_search_regex(r'<title>(.*?)</title>', webpage, 'title')
+        
+        video_url = self._html_search_regex(r'"urls":\["(.*?)"\],', webpage, 'url')
```
Comment (dstftw) [inline youtube_dl/extractor/qianmo.py]: `postInfo` contains far more metadata that should also be extracted. Consider [this](https://github.com/taichatha/youtube-dl/blob/master/youtube_dl/extractor/common.py#L63-L198) for a set of relevant fields.

File: youtube_dl/extractor/qianmo.py (hunk 2 above)
Comment (taichatha) [inline youtube_dl/extractor/qianmo.py]: Sorry, I must have left those in there by accident.

**Comment by yan12125** [comment]
Since #8497 landed, please move changes in ```youtube_dl/extractors/__init__.py``` to ```youtube_dl/extractors/extractors.py```. Check step 5 of the new [developer instructions](https://github.com/rg3/youtube-dl/blob/master/README.md#adding-support-for-a-new-site) for more information.

**Pull request closed without merging**